examples/basics/visuals/box.py

```diff
@@ -10,10 +10,13 @@
 
 from vispy import scene
 from vispy import geometry
+from vispy.visuals.transforms import AffineTransform
 
 canvas = scene.SceneCanvas(keys='interactive', size=(800, 600), show=True)
 
 view = canvas.central_widget.add_view()
+view.bgcolor = '#efefef'
+view.camera = 'turntable'
 
 vertices, faces, outline = geometry.create_box(width=2, height=4, depth=8,
```

Comment:
ouch, yeah. The original also used it (`scene.visuals.box`), so I assumed that it was on purpose. I'll rewrite it as a Visual example